astro/src/layouts/EscapeRoomLayout.astro

```diff
@@ -0,0 +1,470 @@
+---
+import { type PageMetadata } from "../lib/types";
+
+interface Props {
+  metadata?: PageMetadata;
+}
+
+import "../styles/bootstrap.scss";
+
+import Metadata from "../components/Metadata.astro";
+import { Image } from "astro:assets";
+import Branding from "../components/Branding.astro";
+import escapeRoomLogo from "../images/escape-room/escaperoomlogo-768x768.png";
+
+const { metadata } = Astro.props;
+---
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="generator" content={Astro.generator} />
+    <meta name="robots" content="noarchive" />
+
+    {
+      metadata && (
+        <Metadata {...metadata}>
+          <slot name="metadata" slot="metadata" />
+        </Metadata>
+      )
+    }
+    <link rel="icon" type="image/png" href="/images/favicon.png" />
+    <link rel="sitemap" href="/sitemap-index.xml" />
```

Comment:
@brian-montgomery Does it matter if the sitemap is actually at https://accessiblecommunity.org/sitemap-0.xml? i do see the sitemap-index redirects to that url